llvm/tools/llvm-exegesis/lib/BenchmarkRunner.cpp

```diff
@@ -356,7 +356,11 @@ class SubProcessFunctionExecutorImpl
       if (ChildExitCode == 0) {
         // The child exited succesfully, read counter values and return
         // success
-        CounterValues[0] = Counter->read();
+        auto CounterValueOrErr = Counter->readOrError();
+        if (!CounterValueOrErr)
+          return CounterValueOrErr.takeError();
+        CounterValues.swap(*CounterValueOrErr);
```

Comment:
Any reason not to write `CounterValues = *std::move(CounterValueOrErr);` ?

Reply:
Thanks for pointing this out. Updated in the latest revision.